restler/utils/logger.py

```diff
@@ -638,6 +675,71 @@ def log_new_bug():
             os.fsync(log_file.fileno())
 
             return filename
+    
+    def log_new_bug_as_json():
+        # Create the new bug log in json format
+        filename = f"{bucket_class}_{len(bug_buckets[bucket_class].keys())}.json"
```

Comment:
>f"{bucket_class}_{len(bug_buckets[bucket_class].keys())}. [](http://example.com/codeflow?start=19&length=57)

Could you please create a small local function for this, since this is the same logic as for log_new_bug above?  e.g. named "get_bug_filename" - that would help keep these both up to date

Reply:
updated